playbooks/roles/figures/tasks/main.yml

```diff
@@ -56,15 +56,25 @@
     - figures
     - deploy:figures
 
-- name: install figures from {{ figures_code_dir }}
+- name: install figures {{ figures_version }} from {{ figures_code_dir }}
```

Comment:
When did we stop using https://pypi.org/project/Figures/ ? I see `figures_code_dir` so I'm not sure why this is here.

Reply:
@OmarIthawi we used to install figures with `pip` but we discontinued that because the Enterprise theming doesn't render the figures front-end properly and the project owners decided that it we have better things to do than ship the front-end code with the pypi package. So, we decided to install figures from the git repo in the `figures_code_dir`

Reply:
@OmarIthawi we used to install figures with pip but we discontinued that because the Enterprise theming doesn't render the figures front-end properly and the project owners decided that it we have better things to do than ship the front-end code with the pypi package. So, we decided to install figures from the git repo in the figures_code_dir